src/flow.h

```diff
@@ -183,6 +183,16 @@ typedef struct AppLayerParserState_ AppLayerParserState;
         (a)->addr_data32[3] = (p)->ip6h->s_ip6_dst[3];  \
     } while (0)
 
+#define FLOW_SET_IPV4_TTL_FROM_PACKET(p, f) do {    \
```

Comment:
I'm not in favor of these, in part because it confused me when I first looked why these only set toserver

Reply:
Yes, it can be removed now.